test/protocol/call_stack_with_skip_dap_test.rb

```diff
@@ -0,0 +1,467 @@
+# frozen_string_literal: true
+
+require_relative '../support/protocol_test_case'
+
+module DEBUGGER__
+  
+  class CallStackWithSkipDAPTest1664318269 < ProtocolTestCase
```

Comment:
Can this be a request based test? I think we only need to cover the `stackTrace` request for this fix but using recorded test case adds a lot of noise.

You can send a `stackTrace` like [this](https://github.com/ruby/debug/blob/19b4dde3308f532943e4234d1588d4fa26c52345/test/support/protocol_test_case.rb#L176-L180).

Reply:
I went back and forth on this a couple of times. I ended up deciding for this way, because that's what [CONTRIBUTING.md](https://github.com/ruby/debug/blob/master/CONTRIBUTING.md#tests-for-dap-and-cdp) and `ProtocolTestCase` seem to imply, by not making `send_dap_request` part of the API... But 🤷🏽 I agree that this is much better (and probably the docs need to be updated)

Reply:
We were in the process of converting most of the DAP/CDP tests into request-based, so we don't have all request helpers yet.